### PR TITLE
#javascript capstone project

### PR DESCRIPTION
**Pull request by Yemisirach**
we should build these interfaces:
  - The *home page*.
  - The *comments popup*.
  
- we should follow the layout of the wireframes provided. You should personalize the rest of the design including colors, typographies, spacings, etc.
- Home page
  - When the page loads, the webapp retrieves data from:
    - The selected API and shows the list of items on screen.
    - The Involvement API to show the item likes.
  - Remember that your page should make only 2 requests:
    - One to the base API.
    - And **one** to the Involvement API.
  - When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.
  - When the user clicks on the "Comments" button, the Comments popup appears.

  - Home page header and navigation similar to the given mockup.
  - Home page footer similar to the given mockup.
- Comments popup
  - When the popup loads, the webapp retrieves data from:
    - The selected API and shows details about the selected item.
    - The Involvement API to show the item comments.
  - When the user clicks on the "Comment" button, the data is recorded in the Involvement API and the screen is updated.
  - When the popup loads, the webapp retrieves data from:
    - The selected API and shows details about the selected item.
**Counters**
We have counters in all the interfaces that show:
- The number of items (home).
- The number of comments (comments popup).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,162 @@
...
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
Comment (rotimiazeez) [inline README.md]: - The MIT link provided is broken 😢, kindly provide a valid license link 👍🏽.

File: src/Modules/userApi.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  const result = await fetch(`${url}${key}/likes`);
+  const data = await result.json();
+  return data;
+};
+
+const addLike = async (id) => {
+  const body = JSON.stringify({ item_id: id });
+  const headers = { "Content-type": "application/json; charset=UTF-8" };
+  /* eslint-disable */
+
```
Comment (rotimiazeez) [inline src/Modules/userApi.js]: - Please ensure users can like a meal. Currently I'm unable to, kindly fix this 👍🏽

File: src/Modules/getMeal.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+  if (e.target.classList.contains('comment-btn')) {
+    const mealItem = e.target.parentElement.parentElement;
+    fetch(
+      `https://www.themealdb.com/api/json/v1/1/categories.php?i=${mealItem.dataset.id}`,
+    )
+      .then((response) => response.json())
+      .then((data) => mealsFunction(data.categories));
+  }
+}
```
Comment (rotimiazeez) [inline src/Modules/getMeal.js]: Just so you know, when I try to get more information about a meal, I keep getting information about the first food on the list. Please take a look at this; users should be able to see the information on the specific meal they chose. 👍🏽

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- Good UI 👍🏽 
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
- [ ] As a team of 3, you should have a `Reservation` popup that loads. The web app should retrieve data from the selected API and show details about the selected item. And likewise, retrieve data from the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) to show the item reservations.
- [ ] Also, you should have a reserve button, in which when a user clicks on the "Reserve" button, the data should be recorded in the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) and the screen updated as a result.
- [ ] Please ensure counter functions are covered with unit tests using Jest. Make sure that you have tests for all edge cases.

_Check the comments under the review._

Please feel free to use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend considering them as they can improve your code._

You can also consider the following:
- Have a fair workload distribution across the team.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: src/Modules/getMeal.js (hunk 3 above)
Comment (Kingno3l) [reply src/Modules/getMeal.js]: The License link is fixed

File: README.md (hunk 1 above)
Comment (Kingno3l) [reply README.md]: The License link is fixed
